src/components/m-banner-message.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="m-banner-message bg--secondary text--white px-2 py-2">
+    <p>{{ bannerMessage.text }}</p>
+    <i class="el-icon-close" @click="$emit('on-close-banner')"></i>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "m-banner-message",
+  props: {
+    bannerMessage: { type: Object },
```

Comment:
Juste `message` nan ?

```suggestion
    message: { type: Object },
```